cfg/builder/builder_walk.cc

```diff
@@ -58,7 +58,35 @@ LocalRef global2Local(CFGContext cctx, core::SymbolRef what) {
     return alias;
 }
 
-LocalRef unresolvedIdent2Local(CFGContext cctx, const ast::UnresolvedIdent &id) {
+core::ClassOrModuleRef resolveBlockContext(CFGContext cctx, const core::ClassOrModuleRef outterClassOrModule) {
```

Comment:
Nit: this should be `outerClassOrModule` (only one `t` in `outer`) all throughout this change.